### PR TITLE
feature/remove-populate_ntlm_redis_cache

### PR DESCRIPTION
**Pull request by jrouzierinverse**
# Description
Remove the pfcron task populate_ntlm_redis_cache

# Impacts
pfcron

# Delete branch after merge
YES

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries

## Enhancements
* remove pfcron task populate_ntlm_redis_cache

# UPGRADE file entries

### CODE REVIEW

**Comment by jrouzierinverse** [comment]
Still needs the upgrade script to remove the job.

**Comment by nqb** [comment]
@jrouzierinverse, should I consider this PR ready for review ?

**Comment by jrouzierinverse** [comment]
Yes

File: addons/upgrade/to-11.2-pfcron-populate_ntlm_redis_cache.pl [hunk 1]
```diff
@@ -1,36 +1,46 @@
-package pfappserver::Form::Config::Pfcron::populate_ntlm_redis_cache;
+#!/usr/bin/perl
 
 =head1 NAME
 
-pfappserver::Form::Config::Pfcron::populate_ntlm_redis_cache - Web form for populate_ntlm_redis_cache pfcron task
+to-11.2-pfcron-populate_ntlm_redis_cache.p
```
Comment (nqb) [inline addons/upgrade/to-11.2-pfcron-populate_ntlm_redis_cache.pl]:
```suggestion
to-11.2-pfcron-populate_ntlm_redis_cache.pl
```

**Review by nqb — APPROVED** [review]

**Comment by nqb** [comment]
@jrouzierinverse, could you rebase and fix conflict ? Then I will merge.

**Comment by jrouzierinverse** [comment]
done

**Pull request merged**